coop_account/models/account_move_line.py

```diff
@@ -160,7 +158,7 @@ def get_wrong_reconciliation_ml(self):
         for stml in bank_statement_line_ids:
             stml_date = fields.Date.from_string(stml.date)
             move_ids = stml.journal_entry_ids
-            move_line_ids = move_ids.mapped('line_ids')
+            move_line_ids = move_ids
```

Comment:
Hmm.. why? Can you please explain?

Reply:
@ivantodorovich : now we already have move lines

Reply:
it's still not clear to me :thinking:

Reply:
I mean move_ids is an `account.move.line`, while before it was `account.move` so if we do `move_ids.mapped('line_ids')` it give traceback

Reply:
Then you should probably do something like this;

```py
move_line_ids = stml.journal_entry_ids
move_ids = move_line_ids.mapped('move_id')
```

Because if you check below, `move_ids` is referenced and I guess it really expects this.

`if len(move_ids) == 1 and len(move_line_ids) == 2:`

Reply:
@ivantodorovich : done